{
  "modified": "2025-09-26T09:14:03Z",
  "published": "2025-09-26T09:14:03Z",
  "schema_version": "1.5.0",
  "id": "",
  "aliases": [
    "MAL-2025-41688"
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "iamenumerate"
      },
      "versions": [
        "1.0.1"
      ]
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/iamenumerate/MAL-2025-41688.json"
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "b852c726e1c63be7486fee4fcd0a140260ab125b743c70f724b3e84e4f0a4798",
        "import_time": "2025-09-26T11:06:10.151490423Z",
        "id": "RLUA-2025-04775",
        "modified_time": "2025-09-26T09:14:03Z",
        "versions": [
          "1.0.1"
        ]
      }
    ]
  }
}
